test/IRGen/enum.sil

```diff
@@ -1,4 +1,5 @@
-// RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil %s -gnone -emit-ir -disable-objc-attr-requires-foundation-module | %FileCheck %s --check-prefix=CHECK --check-prefix=CHECK-%target-ptrsize --check-prefix=CHECK-%target-runtime-%target-ptrsize
+// Please put -enable-objc-interop tests in enum_objc.sil
```

Comment:
File a SR for this?

Reply:
I'm not following you. I moved the ObjC tests out of `enum.sil` and into `enum_objc.sil` to improve test coverage. Now both Darwin and Linux test *both* "native" and "objc". Before this pull request, Darwin only tested "objc" and Linux only tested "native".

Reply:
Oh, wait, I can't read.  I completely skipped over the "tests" in the comment.  Sorry!